lib/process-services/task-list/components/task-filters.component.ts

```diff
@@ -113,24 +106,11 @@ export class TaskFiltersComponent implements OnInit, OnChanges {
         this.taskFilterService.getTaskListFilters(appId).subscribe(
             (res: FilterRepresentationModel[]) => {
                 if (res.length === 0 && this.isFilterListEmpty()) {
-                    this.taskFilterService.createDefaultFilters(appId).subscribe(
-                        (resDefault: FilterRepresentationModel[]) => {
-                            this.resetFilter();
-                            resDefault.forEach((filter) => {
-                                this.filterObserver.next(filter);
-                            });
-
-                            this.selectTaskFilter(this.filterParam, this.filters);
-                            this.success.emit(resDefault);
-                        },
-                        (errDefault: any) => {
-                            this.error.emit(errDefault);
-                        }
-                    );
+                    this.getDefaultFilters(appId);
                 } else {
                     this.resetFilter();
                     res.forEach((filter) => {
```

Comment:
if the res and filters  are both of them of FilterRepresentationModel[] we don't need the forEach

Reply:
Done.